_posts/2025-03-31-mmaler-blogpost-1-intro.adoc

```diff
@@ -0,0 +1,133 @@
+synopsis: This blog post invites you to discover how Quarkus can help your company cut costs, streamline development, and modernize your Java stack for today’s cloud-native world.
+author: mmaler
+thumbnailimage: /assets/images/posts/mmaler-blog-posts/optimizing-java-with-quarkus-cloud.png
+---
+:imagesdir: /assets/images/posts/mmaler-blog-posts/
+ifdef::env-github,env-browser,env-vscode[:imagesdir: ../assets/images/posts/mmaler-blog-posts]
+
+= Optimizing Java for the Cloud-Native Era with Quarkus
+
+image::optimizing-java-with-quarkus-cloud.png[scaledwidth=100%]
+
+== Intro
+
+Hello! Mickey Maler, a Quarkus Technical Writer, is here.
+
+This short blog post invites you to discover how Quarkus can help your company cut costs, streamline development, and modernize your Java stack for today’s cloud-native world.
+I’ll walk you through real-world benefits, show what Quarkus brings to the table, and open the door to a smoother, faster way of building Java applications—especially in environments where traditional Java frameworks often struggle.
+
+By the end, you might find that switching to Quarkus is easier than you think—and that the support you need to make the move is already within reach.
+Before we explore the details, it's worth noting who is already adopting Quarkus.
+For a quick look at a real-world success story besides a dozen others, let's highlight and mention Orange to start with, a global telecom leader that leveraged Quarkus to achieve its development goals.
+
+* After evaluating multiple frameworks, Orange chose Quarkus as the optimal solution for exposing 5G APIs, thanks to its fast startup, lightweight footprint, modularity, and seamless Kubernetes deployment.
+Quarkus successfully deployed 10 APIs across 4G/5G network cores, with smooth upgrades and optimized resource usage.
+This solidified Quarkus as a key technology for telecom innovation.
++
+For a collection of user stories from the community, see the Quarkus user stories blog series. These stories highlight how different teams and organizations are using Quarkus in the real world.
+
+== What does Quarkus have to offer?
+
+=== Developer joy with live coding and dev mode
+
+Quarkus revolutionizes the development workflow with its live coding feature.
+In traditional Java development, the cycle of writing code, compiling, deploying, and refreshing is time-consuming.
+Quarkus streamlines this process: write your code and refresh the browser—Quarkus instantly detects changes, recompiles, and redeploys.
+This significantly boosts productivity and enhances the developer experience.
+
+=== Versatile build tool integration
+
+Quarkus integrates with popular build tools like Maven and Gradle, making adoption in many existing development workflows easier.
+Maven users benefit from first-class support, while Gradle users can build Quarkus projects with the help of community-maintained tooling, though it currently lacks feature parity with Maven.
```

Comment:
it is both community maintained from context of quarkus.io community blog so I would just leave this out imo